recipes/libheif/all/patches/0001-cmake_1.16.2.patch

```diff
@@ -20,6 +20,18 @@ index de15948..589eab5 100644
  
  option(ENABLE_PLUGIN_LOADING "Support loading of plugins" ON)
  set(PLUGIN_DIRECTORY "${CMAKE_INSTALL_FULL_LIBDIR}/libheif" CACHE STRING "Plugin install directory")
+@@ -74,9 +74,9 @@
+         find_package(${packageName})
+     endif ()
+
+-    if (${variableName}_FOUND AND WITH_${variableName}_PLUGIN AND PLUGIN_LOADING_SUPPORTED_AND_ENABLED)
```

Comment:
Is this then a bug upstream? If so, has it been reported upstream?

Reply:
`variableName` is same as `packageName` upstream.  The patch change the `pacakgeName` to lowercase, which cause `variableName` differ to `packageName`.